src/index.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
     'react/no-direct-mutation-state': 'error',
     'react/no-string-refs': 'error',
     'react/no-unknown-property': 'error',
+    'react/no-unused-prop-types': 'error',
```

Comment:
won't this cause a few false positives with props spreads?
ie, if we do:

```jsx
const Component  = props => <SubComponent {...props} />;

Component.propTypes = {
  foo: PropTypes.string.isRequired
};
```

won't it cause a false positive?

Reply:
Yes it will

Reply:
besides that, if we eventually add propTypes regarding mixins to frontend-clients (so we can have a autocomplete), this will also trigger all kinds of false positives 😅

Reply:
> besides that, if we eventually add propTypes regarding mixins to frontend-clients (so we can have autocomplete), this will also trigger all kinds of false positives 😅

The rule allows passing an `ignore` array so we can skip validation for all those mixin props